gnucash/gnucash-bin.c

```diff
@@ -159,6 +161,14 @@ static GOptionEntry options[] =
            http://developer.gnome.org/doc/API/2.0/glib/glib-Commandline-option-parser.html */
         N_("FILE")
     },
+    {
+        "run-report", '\0', 0, G_OPTION_ARG_STRING, &run_report,
+        N_("Runs a report"), N_("FILE")
+    },
+    {
+        "output-file", '\0', 0, G_OPTION_ARG_STRING, &report_output,
+        N_("Output file for report default gnucash.html"), N_("FILE")
+    },
```

Comment:
Why not printing to stdout by default ? That allows for more scripting flexibility.